instrumentation/spring-boot-actuator-3.0.0/src/main/java/org/springframework/boot/actuate/endpoint/web/servlet/AbstractWebMvcEndpointHandlerMapping_Instrumentation.java

```diff
@@ -0,0 +1,44 @@
+/*
+ *
+ *  * Copyright 2024 New Relic Corporation. All rights reserved.
+ *  * SPDX-License-Identifier: Apache-2.0
+ *
+ */
+package org.springframework.boot.actuate.endpoint.web.servlet;
+
+import com.newrelic.agent.bridge.AgentBridge;
+import com.newrelic.api.agent.NewRelic;
+import com.newrelic.api.agent.Trace;
+import com.newrelic.api.agent.Transaction;
+import com.newrelic.api.agent.TransactionNamePriority;
+import com.newrelic.api.agent.weaver.MatchType;
+import com.newrelic.api.agent.weaver.Weave;
+import com.newrelic.api.agent.weaver.Weaver;
+import com.nr.agent.instrumentation.actuator.SpringActuatorUtils;
+import jakarta.servlet.http.HttpServletRequest;
+
+import java.util.Map;
+
+@Weave(type = MatchType.BaseClass, originalName = "org.springframework.boot.actuate.endpoint.web.servlet.AbstractWebMvcEndpointHandlerMapping")
+public class AbstractWebMvcEndpointHandlerMapping_Instrumentation {
```

Comment:
what about reactive mappings?
org.springframework.boot.actuate.endpoint.web.reactive.AbstractWebFluxEndpointHandlerMapping$ReactiveWebOperationAdapter

Reply:
@zboris   Thanks for the suggestion. I'll get an issue created so it gets on our backlog.